README.md

```diff
@@ -57,6 +57,9 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
     - [Logging](#logging)
     - [Localization](#localization)
     - [Machine Learning](#machine-learning)
+        - [Artificial Intelligence](#artificial-intelligence)
+        - [Personal Assistant](#personal-assistant)
+        - [Miscellaneous](#miscellaneous)
```

Comment:
You mean the following
- [Miscellaneous](#miscellaneous)
should be changed to 
- [Miscellaneous](#miscellaneous-machine-learning)
?

Can you elaborate?